documents/knowledge/yaml_to_js.py

```diff
@@ -48,7 +48,7 @@
                 for overload in element["overloads"]:
                     data_types = map(
                         lambda x: " ".join(x),
-                        zip(overload.split("-"), "abc"),
+                        zip(overload.split("-"), "abcde"),
```

Comment:
A tetrad can have far more than 5 overloads. We should handle that properly instead of just manually adding a letter to this string literal every time an element has too many overloads

Reply:
wdym? this doesn't have to do with how many overloads there is. that doesn't matter. it's just how many arguments there are. so a tetrad has four, so we don't even use the `e`. that would only be used for pentads.

Reply:
Oh, my bad, I should've read it better